src/main/java/core/basesyntax/service/impl/DataConverterImpl.java

```diff
@@ -31,16 +31,16 @@ public List<FruitTransaction> convertToTransactions(List<String> csvLines) {
     }
 
     private int parseQuantity(String quantityStr, String csvLine) {
+        if (Integer.parseInt(quantityStr) < 0) {
```

Comment:
this could be simplified:
`try {
    int quantity = Integer.parseInt(quantityStr);
    if (quantity < 0) {
        throw new IllegalArgumentException("Quantity cannot be less than 0, actual quantity: " + quantity);
    }
} catch (NumberFormatException e) {
    throw new IllegalArgumentException("Invalid quantity format: " + quantityStr, e);
}
`